CMakeLists.txt

```diff
@@ -44,15 +45,9 @@ if (MSVC)
     if (NOT WIN_FLEX_BISON)
         message(FATAL_ERROR "Please install https://github.com/lexxmark/winflexbison and set WIN_FLEX_BISON environment variable. Please use install cmake target after a successful build.")
     endif()
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /std:c++20")
+    set(CMAKE_CXX_STANDARD 20)
 else()
-    if (NOT DEFINED ENV{FLEX_INCLUDE})
-        message(FATAL_ERROR "Please set environment variable FLEX_INCLUDE to the directory containing FlexLexer.h")
-    endif()
-    set(WIN_FLEX_BISON "$ENV{FLEX_INCLUDE}")
-    if (WIN32 AND NOT MINGW)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++20")
-    endif()
+    set(CMAKE_CXX_STANDARD 17)
```

Comment:
Should we instead stick with whatever version is the system default?

Reply:
I think it is better to specify a standard instead of using system default. For me (gcc 12), I test a few options:

* C++20 gives deprecation warning.
* C++14 and 17 work.
* C++11 `src/V3Sched.cpp` gives error like "invalid initialization of reference of type const V3Sched::{anonymous}::EvalKit&".

As a result, I would suggest specifying 14 or 17.

Reply:
No, it is better to generally use the system default, there have been many issues doing otherwise.

Reply:
Agree, AFAIK SystemC requires using a specific C++ standard, let me remove it and push again.